### ISSUE TITLE
Code interpretation Problem

### ISSUE THREAD

**Issue by DOUG888** [winder/Universal-G-Code-Sender#493]
Guys
Update on display UGS Version #654
I have now tried several different POST pros unmodified, HaasTM, HaasVF,Fanuc0M and others and I now get a good graphic EXCEPT for the start partial circle, NB these are unmodified and the graphic result is constant.
Yes,I am doing multiple cuts as I am using a DIODE Laser not a CO2 Laser, adjusted focus is requires, hence the use of a Mill POST.
When I view these Gcodes in other software there is NO partial circle, this leads me to believe there is an odd graphic glitch in UGS Window7.
As before I have attached Code and Picture.
[Files.zip](https://github.com/winder/Universal-G-Code-Sender/files/509621/Files.zip)

I have left the previous comments below if needed.
Thanks 
Doug
*********************************************************************
Previous Entry
i am trying to convert a POST Processor for Bobcam V24, but i keep getting unusual graphic results in UGS Platform.
I am posting here on the off chance that its a graphic issue as I can not see any Gcode error.
I have included all files and 3 pictures to help explain the problem....
Files:- Gcode for parts and Post .
Pictures:-
BAD is what i see with this Gcode.
GOOD is with the Z components excluded (Z is required for multi pass at varying depths to maintain focus)
Also note the partial circle at the ORIGIN.....not in the code at all.
BobCam picture is what i expected to see, this shows a correct tool path.

NB: I have not attempted to run this code as yet.
[Files.zip](https://github.com/winder/Universal-G-Code-Sender/files/505784/Files.zip)
**********************************************************************
Your thoughts.
Thanks 
Doug

**Comment by jahnj0584** [comment]
you may wanna change your post. It does not do the 3/4 circle at the start for me. It DOES move while your Z is down, and then go up the 0.25 before doing its first pass. You also have it as a cycle so it is cutting this part out multiple times in one file.

**Comment by DOUG888** [comment]
My ISSUE has been updated

**Comment by DOUG888** [comment]
OK...May have found the problem.
Line N04 contains a G90 and G54, I removed these and the partial circle disappeared, I tried each one individually, but, it has to be both removed to fix the display. The program run OK as well.
I don't understand why this is so, maybe someone with more experience can answer that.
(N02 T1)
N03 M05
N04 G00 G90 G54 X11.995 Y9.938
N05 M08
N06 Z2.54
N07 Z0.
N08 G01 Z-.25
N09 G03 X11.995 Y9.938 I0. J-2.175 F500.
